__tests__/UrqlConnect_test.re

```diff
@@ -4,34 +4,12 @@ open ReasonUrql;
 Enzyme.configureEnzyme(Enzyme.react_16_adapter());
 
 describe("Connect", () => {
-  module TestQuery = [%graphql
-    {|
-      query dogs {
-        dogs {
-          name
-          breed
-          description
-        }
-      }|}
-  ];
-
-  module TestMutation = [%graphql
-    {|
-      mutation likeDog($key: ID!) {
-        likeDog(key: $key) {
-          name
-          key
-          breed
-          likes
-        }
-      }
-    |}
-  ];
-
-  let testQuery = Query.query(TestQuery.make());
+  let testQuery = Query.query(TestUtils.TestQuery.make());
   let testMutation =
-    Mutation.mutation(TestMutation.make(~key="VmeRTX7j-", ()));
-  let testRender = (result: Connect.renderArgs('a, 'b, 'c)) =>
+    Mutation.mutation(
+      TestUtils.TestMutationWithVariable.make(~key="VmeRTX7j-", ()),
+    );
+  let testRender = (_result: Connect.renderArgs('a, 'b, 'c)) =>
```

Comment:
This is actually the correct syntax here. You can use `~result as _` when you have a named argument but, since the API will keep this variable unnamed, `_result` is correct. http://2ality.com/2017/12/functions-reasonml.html#avoiding-warnings-about-unused-parameters